starwars/src/App.js

```diff
@@ -30,11 +29,14 @@ class App extends Component {
     };
 
     render() {
-        return ( <
-            div className = "App" >
+        return ( < div className = "App" >
             <
-            h1 className = "Header" > React Wars < /h1> < /
-            div >
+            h1 className = "Header" > React Wars < /h1>     <
+            CharacterList characters = { this.state.starwarsChars }
+            cars = { this.state.starwarsChars.vehicles }
```

Comment:
This won't work because this.starwarsChars is an array, not an object. Regardless, you don't need this line here.